.github/workflows/lint-pr.yml

```diff
@@ -19,6 +19,17 @@ jobs:
         with:
           egress-policy: audit # TODO: change to 'egress-policy: block' after couple of runs
       - uses: amannn/action-semantic-pull-request@01d5fd8a8ebb9aafe902c40c53f0f4744f7381eb # v5.0.2
+        with:
+          # Configure additional validation for the subject based on a regex.
+          # This example ensures the subject doesn't start with an uppercase character.
```

Comment:
simplify the text to say `Ensure subject doesn't start with an uppercase character`?

Reply:
Thanks, updated. PTAL!